src/UserMovieCard/UserMovieCard.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+// import './MovieCard.css';
+
+const UserMovieCard = ({ title, averageRating, poster, id, key, usersRatings }) => {
```

Comment:
So we are passing in usersRatings into every card? Maybe there is a way we could use find and only send it in with the users rating for that particular movie